### PR TITLE
fix(components/Virtualized): adapt the toolbar stories

### PR DESCRIPTION
**Pull request by jare-talend**
**What is the problem this PR is trying to solve?**
Autosizer takes the height from her parent. https://github.com/bvaughn/react-virtualized/blob/master/docs/usingAutoSizer.md

With the toolbar is enable, the list height is the toolbar height + display list height instead of only the list height. 
To avoid to get a height of the list higher, we need to set a wrapper around the autosize to determinate the height.

**What is the chosen solution to this problem?**

**Please check if the PR fulfills these requirements**
- [ ] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/components/src/List/List.scss [hunk 1]
```diff
@@ -19,5 +19,7 @@
 		box-shadow: $shadow-default;
 	}
 
-
+	:global(.tc-list-display-virtualized) {
+		overflow-y: hidden;
```
Comment (jsomsanith) [inline packages/components/src/List/List.scss]: This is not a good solution. This hides a scrolling issue du to css placement.

File: packages/components/src/List/List.component.js [hunk 2]
```diff
@@ -54,12 +54,18 @@ function DisplayModeComponent({ id, useContent, displayMode, list, virtualized }
 		);
 	}
 	if (virtualized) {
+		const classnames = classNames(
+			'tc-list-display',
+			'tc-list-display-virtualized',
```
Comment (jsomsanith) [inline packages/components/src/List/List.component.js]: This is not needed, only `tc-list-display` is

File: packages/components/stories/List.js [hunk 3]
```diff
@@ -600,4 +600,34 @@ storiesOf('List', module)
...
+		inputDebounceProps.toolbar.filter.debounceTimeout = 300;
+
+		return (<div style={{ height: '60vh' }} className="virtualized-list">
+			<IconsProvider />
+
+			<h1>List</h1>
+			<h2>Definition</h2>
+			<p>Toolbar Filter</p>
+			<h2>Docked</h2>
+			<List {...dockedProps} virtualized />
```
Comment (jsomsanith) [inline packages/components/stories/List.js]: To avoid the list to extends its size, let's wrap each list in a controlled size div

File: packages/components/stories/VirtualizedList.js [hunk 4]
```diff
@@ -188,7 +188,7 @@ for (let i = collection.length; i < 100; i += 1) {
 
 storiesOf('Virtualized List', module)
 	.add('List > Table', () => (
-		<div style={{ height: '60vh' }} className="virtualized-list">
+		<div className="virtualized-list">
```
Comment (jsomsanith) [inline packages/components/stories/VirtualizedList.js]: Why the changes in this file ?

File: packages/components/src/List/List.component.js (hunk 2 above)
Comment (jare-talend) [inline packages/components/src/List/List.component.js]: I have added tc-list-display-virtualized to avoid to override overflow-y: hidden; on tc-list-display during the migration

File: packages/components/stories/VirtualizedList.js (hunk 4 above)
Comment (jare-talend) [inline packages/components/stories/VirtualizedList.js]: I wrapped the list into <section style={{ height: '300px'}}></section>. It is unused to define a height.

File: packages/components/src/List/List.component.js (hunk 2 above)
Comment (jsomsanith) [inline packages/components/src/List/List.component.js]: Yes but `overflow-y: hidden;` is not a good solution (see above)

File: packages/components/stories/VirtualizedList.js (hunk 4 above)
Comment (jsomsanith) [inline packages/components/stories/VirtualizedList.js]: To avoid List to take the h1 and p size :+1:
